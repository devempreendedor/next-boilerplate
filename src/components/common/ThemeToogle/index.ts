export { default as ThemeToogle } from './ThemeToogle'
