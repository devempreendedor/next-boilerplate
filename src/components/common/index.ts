export * from './ThemeToogle'
export * from './Layout'
