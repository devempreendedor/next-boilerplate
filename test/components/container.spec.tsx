import { render, screen } from '../test-utils'
import { Container } from '@components/common'

describe('Container Component', () => {
  it('Should render the container', () => {
    render(
      <Container>
        <h1>Container</h1>
      </Container>
    )

    const container = screen.getByText(/Container/i)

    expect(container).toBeInTheDocument()
    expect(container).toMatchSnapshot()
  })
})
