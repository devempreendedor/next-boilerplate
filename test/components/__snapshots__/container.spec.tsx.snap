// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Component Should render the container 1`] = `
<h1>
  Container
</h1>
`;
